lang/zh_TW.lang

```diff
@@ -580,14 +729,14 @@ option.MOTION_BLUR.comment=啟用基於相機運動的模糊。
 option.MOTION_BLUR_STRENGTH=運動模糊強度
 option.MOTION_BLUR_STRENGTH.comment=調整運動模糊的強度。
 
-option.BLOOM=泛光
-option.BLOOM.comment=允許亮區向外發光。
+option.BLOOM=光暈
```

Comment:
looking at google translate, old is "bloom" while the new one is just "halo". please recheck.